packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -320,6 +320,10 @@ export async function getAvailableSlots({ input, ctx }: GetScheduleOptions) {
     usersWithCredentials = eventType.hosts.map(({ isFixed, user }) => ({ isFixed, ...user }));
   }
 
+  if (eventType.schedulingType && !eventType.hosts?.length) {
```

Comment:
I think we need to check for `eventType.users` too, I think older event types can have set users instead of hosts. We also take them into account in line 314

Reply:
I just talked to @emrysal, we should still support `eventtype.users`. So we need to fix the error when booking `"Cannot read properties of undefined (reading 'name')".`

Reply:
i am not able to find this error message. I don't see anything in console.log and booking is created in DB and "Cannot read properties of undefined (reading 'name') messages is displayed on booking page.